protocol/tbdex_schema_examples.js

```diff
@@ -0,0 +1,50 @@
+let offering = {
```

Comment:
overall comment - still undecided on if we need ids for each message type, and if so, what kinds of ids, or maybe a single transactionId across all message types?

Reply:
`trace_id` I think is my vote... it's a concept which could be extended to other web5 protocols